### PR TITLE
Have the same window.width keys as everywhere else

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: extension/background.js [hunk 1]
```diff
@@ -160,8 +160,8 @@ function sendData(name, action) {
 		'url' : data.url,
 		'os' : data.os,
 		'browser' : data.browser,
```
Comment (beatngu13) [inline extension/background.js:162]: What about these? Compared to:

https://github.com/retest/recheck-web/blob/6b9617670bc135a3e08cea8c0e9dc7aa41d90204/src/main/java/de/retest/web/meta/SeleniumMetadata.java

Not relevant here, but maybe we should also change the class name to `WebMetadata` as not everything is Selenium-specific.

Comment (roesslerj) [reply extension/background.js:162]: A bit more effort as browser currently is a summary of browser name and version. Same for os.

Comment (beatngu13) [reply extension/background.js:162]: Ah, OK. Thx for explaining.

**Review by beatngu13 — APPROVED** [review]

File: extension/background.js (hunk 1 above)
Comment (roesslerj) [reply extension/background.js:162]: Changing the class name is a good idea. But not relevant before the release, right? It's internal anyways...

Comment (beatngu13) [reply extension/background.js:162]: Yes, was just a note. Nothing important.

**Pull request merged**